docs/server/source/production-deployment-template/add-node-on-kubernetes.rst

```diff
@@ -88,12 +144,17 @@ will have to coordinate offline with an existing administrator so that they can
 add the new MongoDB instance to the replica set.
 
 Add the new instance of MongoDB from an existing instance by accessing the
-``mongo`` shell.
+``mongo`` shell and authenticate as the ``adminUser`` we created for existing MongoDB instance:
```

Comment:
Mention that the existing MongoDB node should be a PRIMARY? Or contact the admin of the node that is PRIMARY?